packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -209,12 +209,14 @@ export class PharosImageCard extends ScopedRegistryMixin(FocusMixin(PharosElemen
 
   private _renderCollectionImage(): TemplateResult {
     return html`<div
+      class="card__link--container"
```

Comment:
`--container` isn't really serving as a modifier here, by BEM standards. I think `class="card-link-container"` is probably okay here.